host/FirecrackerSession.ts

```diff
@@ -102,6 +102,13 @@ export class FirecrackerSession extends Session {
     this.bootArgs = 'reboot=k panic=1 pci=off'
   }
 
+  repr(): any {
+    const { id } = this
+    return {
+      id
+    }
+  }
+
```

Comment:
Why not just:
`return { id: this.id }` ?

Reply:
In this case, no reason. I've just been finding destructuring assignments really useful and concise in other situations (e.g. providing a default value) and was carrying on with the idiom.